base/linalg/eigen.jl

```diff
@@ -54,6 +54,27 @@ function eigfact!{T<:BlasComplex}(A::StridedMatrix{T}; permute::Bool=true, scale
     ishermitian(A) && return eigfact!(Hermitian(A))
     return Eigen(LAPACK.geevx!(permute ? (scale ? 'B' : 'P') : (scale ? 'S' : 'N'), 'N', 'V', 'N', A)[[2,4]]...)
 end
+
+"""
+    eigfact(A,[irange,][vl,][vu,][permute=true,][scale=true]) -> Eigen
+
+Computes the eigenvalue decomposition of `A`, returning an [`Eigen`](:obj:`Eigen`) factorization object `F`
+which contains the eigenvalues in `F[:values]` and the eigenvectors in the columns of the
+matrix `F[:vectors]`. (The `k`th eigenvector can be obtained from the slice `F[:vectors][:, k]`.)
+
+The following functions are available for `Eigen` objects: [`inv`](:func:`inv`), [`det`](:func:`det`), [`isposdef`](:func:`isposdef`), .
```

Comment:
This line ends with `, .`. Should there be another function in that list or is it just a missing `and` at the end?